.github/workflows/ci.yml

```
@@ -171,7 +171,7 @@ jobs:
           qiskit_version=$(sed qiskit-superstaq/requirements.txt -ne 's/^qiskit.=/qiskit==/p')
           echo "Using minimum qiskit version: $qiskit_version"
           python -m pip install --upgrade pip
-          pip install "$qiskit_version" -e ./general-superstaq[dev] -e ./qiskit-superstaq[dev]
+          pip install "$qiskit_version" -e ./checks-superstaq -e ./qiskit-superstaq[dev]
```

gss is still necessary here
```suggestion
          pip install "$qiskit_version" -e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[dev]
```